processing/src/test/java/org/apache/druid/collections/spatial/SpatialUtils.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.collections.spatial;
+
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class SpatialUtils
+{
+  private static Random random = ThreadLocalRandom.current();
+
+  public static float[][] generateGeoCoordinatesAroundCircle(
```

Comment:
since its only used in test, the method should be moved to test code

Reply:
it is already in test package.